local/gugcat/renderer.php

```diff
@@ -177,13 +178,16 @@ public function display_aggregation_tool($rows, $activities) {
             $htmlrows .= html_writer::tag('td', $row->studentno);
             $htmlrows .= html_writer::tag('td', $row->surname);
             $htmlrows .= html_writer::tag('td', $row->forename);
-            foreach((array) $row->grades as $grdobj) {
-                $htmlrows .= '<td>'.$grdobj->grade.((strpos($grdobj->grade, 'No grade') !== false) ? null : $this->context_actions($row->studentno)).'</td>';
+
+            foreach((array) $row->grades as $grade) {
+                echo $grade;
```

Comment:
Remove echo $grade die;

Reply:
<img width="816" alt="image" src="https://user-images.githubusercontent.com/74400168/104009599-4c568a80-51e6-11eb-901f-8615c70aa9bd.PNG">

Reply:
I've already fix that marj